metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_query.py

```diff
@@ -940,3 +951,37 @@ def streams_for_database(
             f"""FROM '{stream_pagination_marker}'""" if stream_pagination_marker else ""
         )
         return f"""SHOW STREAMS IN DATABASE "{db_name}" LIMIT {limit} {from_clause};"""
+
+    @staticmethod
+    def show_dynamic_tables_for_database(
+        db_name: str,
+        limit: int = SHOW_VIEWS_MAX_PAGE_SIZE,
```

Comment:
Since this constant is reused, it may be renamed as `SHOW_MAX_PAGE_SIZE` or `SHOW_COMMAND_MAX_PAGE_SIZE`